src/app/components/footer/footer.component.html

```diff
@@ -1,36 +1,47 @@
 <footer class="pb-0 pt-2 bg-light">
   <div class="container">
-    <div class="row justify-content-between">
-      <div class="col-md-4">
-          <span placement="top" [ngbTooltip]="tipContent" i18n tabindex="0">Program versions</span>
-      </div>
-      <div class="col-md-5 pull-right">
-        <div class="row">
-          <div class="col-md-6">
+    <div class="col-md-12">
+      <nav class="row justify-content-between">
+        <ul class="row">
+          <li class="col-md">
+            <span placement="top" [ngbTooltip]="tipContent" i18n tabindex="0">Program versions</span>
+          </li>
+          <li class="col-md-auto">
             <i class="fa fa-envelope-o" aria-hidden="true"></i>
             <a class="text-dark" href="mailto:{{contactAddress}}">{{contactAddress}}</a>
-          </div>
-          <div class="col-md-4">
-            <i class="fa fa-github" aria-hidden="true"></i>
-            <a class="text-dark" href="https://github.com/zonemaster/zonemaster">Github</a>
-          </div>
+          </li>
+          <li class="col-md-auto">
+            <i class="fa fa-code" aria-hidden="true"></i>
+            <a class="text-dark" href="https://github.com/zonemaster/zonemaster">Source code</a>
+          </li>
+          <li class="col-md-auto">
+            <i class="fa fa-book" aria-hidden="true"></i>
+            <a class="text-dark" href="https://github.com/zonemaster/zonemaster#documentation">Documentation</a>
+          </li>
+        </ul>
+      </nav>
+
+      <div class="row">
+        <div class="col-md-auto">
+          <p class="copyright small" i18n>Copyright © AFNIC and The Swedish Internet Foundation. All Rights Reserved</p>
         </div>
       </div>
+
     </div>
 
-    <div class="row">
-      <div class="col-md-12">
-        <p class="copyright text-muted small" i18n>Copyright © AFNIC and The Swedish Internet Foundation. All Rights Reserved</p>
-      </div>
+    <div class="col-md-12" *ngIf="footerLogo !== ''">
```

Comment:
Is it the wanted behaviour to have the logo below the footer? Can you provide a example how this is used?

I don't mind  having a optional logo in the footer I just want to understand how it integrates with the current UI if someone else wants to use it.

Reply:
We will use CSS to place and size the logotype in a good locatioin. We will replace the Zonemaster logo in the top with a Internetstiftelse logo in the top and put an Afnic logo in the footer.

Reply:
> We will use CSS to place and size the logotype in a good locatioin.

In this case it may be good to have some default CSS rules to prevent oversize and resize the logo if needed on smaller screen included in the pull request so that someone that want to use the feature does not have to hack the GUI to make it work. What do you think?

Example when I used the (clearly oversized) Zonemaster logo:
![Screenshot 2023-05-09 at 17-02-16 Zonemaster](https://github.com/zonemaster/zonemaster-gui/assets/19394895/328d371a-def1-4891-ac34-58609a5208ad)

Reply:
> > We will use CSS to place and size the logotype in a good locatioin.
> 
> In this case it may be good to have some default CSS rules to prevent oversize and resize the logo if needed on smaller screen included in the pull request so that someone that want to use the feature does not have to hack the GUI to make it work. What do you think?
> 
> Example when I used the (clearly oversized) Zonemaster logo: ![Screenshot 2023-05-09 at 17-02-16 Zonemaster](https://user-images.githubusercontent.com/19394895/237137713-328d371a-def1-4891-ac34-58609a5208ad.png)

Yeah, that's probably a good idea. Let's add a class e.g `footer-logotype` to the image and som default styles like

```
.footer-logotype {
    width: 100%;
    max-width: 150px;
    height: auto;
}
```

Reply:
Thanks.

I was wondering, I don't see the contact address in your screenshot, do you hide it in CSS? If that is the case, maybe we should make it so that if it is empty in the configuration it is not rendered in the template.

Reply:
I have set class to the footer logotype.